content/en/docs/setup/independent/kubelet-integration.md

```diff
@@ -8,113 +8,160 @@ weight: 40
-the `.kubeletConfiguration.baseConfig` key. This makes it possible for any user writing a `MasterConfiguration`
-file to also set all kubelets' base-level configuration.
+### Workflow when using `kubeadm init`
 
-When `kubeadm init` is called, the `.kubeletConfiguration.baseConfig`  structure is marshalled to disk, to the
-following path - `/var/lib/kubelet/config.yaml`. This marshalled content is also uploaded to a ConfigMap in the cluster
-named `kubelet-config-1.X`, where `X` is the minor version of the Kubernetes version initialized. `kubeadm init` also
-generates the `/etc/kubernetes/kubelet.conf` KubeConfig file with credentials (client certificates) so that the kubelet
-can talk to the API server.
+When you call `kubeadm init`, the `.kubeletConfiguration.baseConfig`  structure is marshalled to disk
+at `/var/lib/kubelet/config.yaml`, and also uploaded to a ConfigMap in the cluster. The ConfigMap
+is named `kubelet-config-1.X`, where `.X` is the minor version of the Kubernetes version you are
+initializing. A kubelet configuration file is also written to `/etc/kubernetes/kubelet.conf` with the
+baseline cluster-wide configuration for all kubelets in the cluster. This configuration file
+points to the client certificates that allow the kubelet to communicate with the API server. This
+addresses the need to
+[propogate cluster-level configuration to each kubelet](#propagating-cluster-level-configuration-to-each-kubelet).
+
+To address the second pattern of
+[providing instance-specific configuration details](#providing-instance-specific-configuration-details),
+kubeadm writes an environment file to `/var/lib/kubelet/kubeadm-flags.env`, which contains a list of
+flags to pass to the kubelet when it starts. The flags are presented in the file like this:
 
-In order to address _Problem 2_, `kubeadm init` writes an environment file looking like this, in the path
-`/var/lib/kubelet/kubeadm-flags.env`:
 ```bash
 KUBELET_KUBEADM_ARGS="--flag1=value1 --flag2=value2 ..."
 ```
 
-kubeadm also writes to the file dynamic parameters like what cgroup driver that is used, or whether a non-docker
-`--cri-socket` is used. Then kubeadm attempts a `systemctl daemon-reload && systemctl restart kubelet` in order
-to make the kubelet pick up the latest dynamic flags which were previously written. After that the rest of the regular
-`kubeadm init` workflow continues.
+In addition to the flags used when starting the kubelet, the file also contains dynamic
+parameters such as the cgroup driver and whether to use a different CRI runtime socket
+(`--cri-socket`).
 
-On the other hand, `kubeadm join` uses the Bootstrap Token credential to download the `kubelet-config-1.X` ConfigMap
-and write it to `/var/lib/kubelet/config.yaml`.
+After marshalling these two files to disk, kubeadm attempts to run the following two
+commands, if you are using systemd to manage your processes:
```

Comment:
through out the doc, "process manager" and "manage a process" is slightly misused in the case of systemd, because it's more of a "service manager".
a process is not exactly the same as a service in any OS context.

i would omit ` to manage your processes` to dodge this ambiguity.